src/librustc_typeck/diagnostics.rs

```diff
@@ -1972,6 +1972,88 @@ To learn more about traits, take a look at the Book:
 https://doc.rust-lang.org/book/traits.html
 "##,
 
+E0174: r##"
+This error occurs because of unboxed closure explicitly methods calls is
```

Comment:
I'm going change to the original short description error:  "explicit use of unboxed closure methods are experimental"